src/metrics/loc.rs

```diff
@@ -480,28 +480,47 @@ mod tests {
 
         // LLOC returns three because there is an empty Rust statement
         check_metrics!(
-            "let a = 42;\n
-             if true {\n
-                42\n
-             } else {\n
-                43\n
+            "let a = 42;
+             if true {
+                42
+             } else {
+                43
              };\n",
             "foo.rs",
             RustParser,
             loc,
             [(lloc, 3, usize)]
         );
-    }
 
-    #[test]
-    fn test_lloc_c() {
         check_metrics!(
-            "for (;;)\n
+            "for (;;)
                 break;\n",
             "foo.c",
             CppParser,
             loc,
             [(lloc, 2, usize)]
         );
     }
+
+    #[test]
+    fn test_general_loc() {
```

Comment:
Since this is a new test and is not part of the refactoring of the pre-existing tests, it should be part of a separate PR